be/src/exec/table_function_node.cpp

```diff
@@ -80,7 +80,6 @@ Status TableFunctionNode::init(const TPlanNode& tnode, RuntimeState* state) {
         return Status::InternalError("can't find table function " + table_function_name);
```

Comment:
No need to update the non-pipeline engine.

Reply:
Need to update the non-pipeline engine because it does not support handle the same input row multiple times.